tests/e2e/wp/create.spec.ts

```diff
@@ -210,4 +210,18 @@ test.describe("Creating a new Work Plan", () => {
     await modal.getByRole("button", { name: "Return to dashboard" }).click();
     await expect(unlockButton).toBeDisabled();
   });
+
+  test("State user can resubmit a work plan", async () => {
+    await wpDashboard.goto();
+    await wpDashboard.firstReport.getByRole("button", { name: "Edit" }).click();
```

Comment:
Do we care about writing out the selector in this file vs in `wpDashboard.page.ts`?

Reply:
In general I would lean toward only including selectors in the page object if we know they are going to be used more than once. I see that we are actually using the Edit button here twice, so it could be a candidate to move. I'll follow up on that!